pyanaconda/ui/gui/spokes/custom_storage.py

```diff
@@ -1458,15 +1471,17 @@ def _setup_device_type_combo(self, device, device_name):
 
             self._device_name_dict[_type] = name
 
-        itr = self._typeStore.get_iter_first()
-        while itr:
-            if dev_type_from_const(self._typeStore[itr][1]) == device_type:
+        if not self._set_device_type(device_type):
+            unsupported = device_type in (dev_type_from_const(dc) for dc
+                                          in should_appear - should_appear_supported)
```

Comment:
For better readability I would like to see that unsupported list creation in a function or property. It could be even generator.

Or you can create function `is_unsupported_device_type` which is complement of `is_supported_device_type` function.

Reply:
Hm, it doesn't make sense to me. Did you see how should_appear and should_appear_supported is created above? I really can't imagine how such a function or property would look like and how it would be useful. Any suggestion? I don't think it is less readable then the rest of the code. Making this look better would require bigger rewrite of the function and related code and I didn't want to do it in this PR.

`Or you can create function is_unsupported_device_type which is complement of is_supported_device_type function.`
Why? I could use not is_supported_device_type (moreover this is a function from blivet). Still I don't see how should I use it to make things better.

Reply:
Personally I would abstract all the `should_appear_supported`, `should_appear` and so on, because it is just a noise in this method and in general make this method over-complicated.

But I understand that this is not purpose of the PR and I'm not insisting on this change.